.semaphore/semaphore.yml

```diff
@@ -33,11 +37,22 @@ blocks:
             - "sh ./install-nix-2.3.15 --no-daemon"
             - ". $HOME/.nix-profile/etc/profile.d/nix.sh"
 
+            # Install Cachix and use the Channable cache
+            - "nix-env -iA nixpkgs.cachix"
+            - "cachix use channable-public"
+
+            # Build the devenv and the static package and upload to Cachix
+            - "nix-build --no-out-link default.nix nix/release.nix | cachix push channable-public"
```

Comment:
What happens here when `nix-build` fails does the semaphore job still succeed? We had such an issue recently in Hoff. That was fixed by splitting those combined steps into separate steps, see https://github.com/channable/hoff/pull/120/files

Reply:
The pipeline would still fail as the next commands also use the same builds and would fail, but this is merely by coincidence.
I'll change the pipe to a redirection.